pkg/gen/python-templates/CustomResource.py

```diff
@@ -0,0 +1,49 @@
+# *** WARNING: this file was generated by the Pulumi Kubernetes codegen tool. ***
+# *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+import pulumi
+import pulumi.runtime
+
+from .. import tables
+
+
+class CustomResource(pulumi.CustomResource):
```

Comment:
Any ideas on alternative names to avoid clashes with `pulumi.CustomResource` in conversation?

Reply:
Not really. We already have precedent with the NodeJS SDK and the way all of the other k8s resources are named. It's unfortunate, though.